Source/santa/SNTNotificationManager.m

```diff
@@ -112,12 +114,49 @@ - (void)queueMessage:(SNTMessageWindowController *)pendingMsg {
 
   pendingMsg.delegate = self;
   [self.pendingNotifications addObject:pendingMsg];
+  [self postDistributedNotification:pendingMsg];
```

Comment:
Should whether or not we send notifications be configurable?

Reply:
I don't think so, the notifications are relatively cheap (esp. in the context of only sending them for de-duped blocked executions) and there's nothing sensitive in the notifications that can't easily be gleaned from other sources.